### PR TITLE
DTSPO-12396 - install Openjdk 17 on non-prod bastion

### PR DESCRIPTION
**Pull request by willwatters**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by kunlebalogun — APPROVED** [review]

File: provision-bastion.sh [hunk 1]
```diff
@@ -12,7 +12,7 @@ echo "deb [signed-by=/usr/share/keyrings/kubernetes-archive-keyring.gpg] https:/
 apt-get update
 apt-get upgrade -y
 
-apt-get -y install apt-transport-https azure-cli ca-certificates curl gnupg jq kubectl lsb-release nmap openjdk-11-jre-headless postgresql tcpdump parallel redis-server
+apt-get -y install apt-transport-https azure-cli ca-certificates curl gnupg jq kubectl lsb-release nmap openjdk-17-jre-headless postgresql tcpdump parallel redis-server
```
Comment (msl8r) [inline provision-bastion.sh:15]: is anyone using openjdk-11? removing this could cause issues for them

Comment (timja) [reply provision-bastion.sh:15]: probably safer to have both if you can for now

Comment (kunlebalogun) [reply provision-bastion.sh:15]: Good idea

Comment (willwatters) [reply provision-bastion.sh]: I have updated the PR to include both openjdk-11-jre-headless & openjdk-17-jre-headless

**Pull request merged**